package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "rosnodejs",
-  "version": "3.0.2",
+  "name": "@rethink-robotics/rosnodejs",
```

Comment:
@chfritz do you have any other solution to this?!
@chris-smith you seem to own rosnodejs at npmjs, can you do anything about it?!

Reply:
The only option for keeping the rosnodejs name is to have @chris-smith publish a new version. @chris-smith, feel free to add me as a maintainer on npm if you want to hand off some of this more basic maintenance.
@DasRoteSkelett if you urgently need the latest version, then I would recommend you just use the github url for rosnodejs itself in your project, i.e.:
```json
{ "dependencies": {
    "rosnodejs": "github:RethinkRobotics-opensource/rosnodejs"
  }
}
```

Reply:
Yes I'm not aware that you can use a different name in the `package.json` vs in npm. I think you'd either need to revert the name change or you'd have to publish a new package with a different name.

Obviously you are free to do so though I wouldn't recommend it. If the goal is just to get a new release published there's no reason to do so.

Reply:
well, @chris-smith , since you own rosnodejs at npmjs.com, you could either publish a new version there, or, transfer the right to publish there to someone who wants to step up and maintain this. Then my pull request is obsolete, of course.
![image](https://user-images.githubusercontent.com/6168467/160564192-f10e7e00-b327-4fcf-860e-77284aeda5b2.png)
The stuff with the @rethink-robotics/rosnodejs is just the workaround for the case you got hit by a bus or moved off to some other challenges...

Reply:
Happy to make a new release. Would you like to update this PR or should I open a new one?

Reply:
I will reuse this one. Version 3.0.3 okay, or would you prefer 3.1.0?

Reply:
I think `3.1.0` makes more sense looking at the commits that have gone in.

Reply:
@chris-smith Done rebasing.